src/playbacks/html5_video/html5_video.js

```diff
@@ -251,11 +253,11 @@ export default class HTML5Video extends Playback {
   }
 
   mute() {
```

Comment:
it's unclear to me: are these methods even used somewhere? :)

they aren't part of the Playback interface definition, correct?

i am aware that in fact Clappr shares concept of "muted" and "volume === 0". which i think is a good thing to do in order to keep things actually clearer and simpler for applications.

here in the case of HTML5 playback, there is this native property, and it is important to set it as configured before first play() in order to allow auto-play on platforms that only allow that when it is muted (not necessarily native volume property zero but actually muted and defaultMuted set).